qb/update.go

```diff
@@ -0,0 +1,105 @@
+package qb
+
+import (
+	"bytes"
+)
+
+// UPDATE reference:
+// http://docs.datastax.com/en/dse/5.1/cql/cql/cql_reference/cql_commands/cqlUpdate.html
```

Comment:
I think we should point to Apache Cassandra instead of the vendor specific ones:

https://cassandra.apache.org/doc/latest/cql/dml.html#update